mod/admin.php

```diff
@@ -784,10 +784,7 @@ function admin_page_workerqueue(App $a)
 	$statement = DBA::select('workerqueue', ['id', 'parameter', 'created', 'priority'], ['done' => 0], ['order'=> ['priority']]);
 	$r = DBA::inArray($statement);
 
-	for($i = 0; $i < count($r); $i++) {
-		$r[$i]['parameter'] = stripslashes(implode(': ', explode('","', $r[$i]['parameter'])));
-		$r[$i]['parameter'] = substr($r[$i]['parameter'], 2, -2);
- 	}
+        $r[$i]['parameter'] = implode(json_decode($r[$i]['parameter']), ': ');
```

Comment:
Idea is good, indentation is wrong (should be a tab)

Reply:
Should fix this in my IDE, but I have no idea where ;)

Using netbeans :)

Reply:
Right-click on Project -> Properties -> Formatting -> Use project specific options -> Language: PHP -> Category: Tabs and indent.